src/views/workflow-history/helpers/__tests__/workflow-history-fetcher.test.tsx

```diff
@@ -262,6 +262,37 @@ describe(WorkflowHistoryFetcher.name, () => {
     const state = fetcher.getCurrentState();
     expect(state.data?.pages.length).toBe(pageCountBefore);
   });
+
+  it('should use WORKFLOW_HISTORY_FIRST_PAGE_SIZE_CONFIG for the first page', async () => {
+    const { fetcher, getCapturedPageSizes } = setup(queryClient);
+
+    fetcher.start((state) => !state?.data?.pages?.length);
+
+    await waitFor(() => {
+      const state = fetcher.getCurrentState();
+      expect(state.data?.pages).toHaveLength(1);
+    });
+
+    const pageSizes = getCapturedPageSizes();
+    expect(pageSizes).toHaveLength(1);
+    expect(pageSizes[0]).toBe('200');
```

Comment:
This seems to assert behavior of `setup` and `mockEndpoints` rather than the "unit under test".

Is this intended?